### ISSUE TITLE
Source code, will it be open source?

### ISSUE THREAD

**Issue by avelino** [pixelfed/mobile-app#31]
In the repository there is no code, will the mobile application be open source ?

**Comment by pylapp** [comment]
In addition it could be very nice to provide the source code publicly so as to get contributors. I see some points to improve on the iOS part at least and would like to help, but the source code is still hidden 😭

**Comment by pylapp** [comment]
@dansup When do you plan to release the source code? It will be great and some developers could help 😇

**Comment by pylapp** [comment]
@dansup Just found some bugs in the iOS app ; having it open source will be appreciated and I will be able to bring the fixes

**Comment by Saimyx** [comment]
I agree, I won't trust this app as long as its source code is closed.
And that's a pity, open-source and free license are part of the core of the Fediverse tools.

**Comment by Moonstone3010** [comment]
There is info on [@Pixelfed](https://mastodon.social/@Pixelfed) that apps will appear on the App Store and Play Store at the end of May. There is no information on publishing the source so maybe they won't be open source.

**Comment by HopperMCS** [comment]
Just want to chime in that I don't run closed source code on my servers.

**Comment by saghul** [comment]
This is the mobile app repository. Pixelfed is open source already, you can run it on your own server.

**Comment by saghul** [comment]
My reply was to @HopperMCS

**Comment by pylapp** [comment]
Up 🫶

BTW feel free all to support Pixelfed, I really enjoy using this awesome project!

https://pixelfed.org/support-our-project
